### DISCUSSION TITLE
Sketchpad/whiteboard feature

### DISCUSSION THREAD

**Discussion by enriquesanchez** [ParabolInc/parabol#5485] [New features]
From the [Parabol Customer Feedback Survey](https://parabol.slack.com/archives/C08FL5UE7/p1633422567280700) 🔒, users expressed particular enthusiasm for a sketchpad/whiteboard feature.

This has also come up before (see #4110 and #1941).

While more focused on the icrebreaker phase, [we've explored some related ideas here](https://www.notion.so/parabol/Icebreaker-v2-4518fefec5574a1f8c9c40d1ec18a8d2#4e153ae7d1ad4575b015e5e52a502f94) 🔒.

**Comment by mattkrick** [comment]
see #5659
